docs/notebooks/01_merge.ipynb

```diff
@@ -1,27 +1,505 @@
 {
```

Comment:
<p>I tested <code>ps.pp.aggregate_duplicate_gene_ids(d) on a few toy examples. Hopefully it does what I expect in all cases.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icbi-lab/atlas_protocol/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>